data/speakers.ts

```diff
@@ -339,6 +339,24 @@ const speakers: Speaker[] = [
     readyToTravelAtOwnExpense: null,
     currentLocation: { continent: 'Africa', country: 'Nigeria', city: 'Lagos' },
   },
+  {
+    objectID: 'realbakari',
+    fullName: 'Bakari Mustafa',
+    tags: ['JavaScript', 'Python', 'JavaScript'],
+    email: 'contact@bakarimustafa.com',
+    website: 'https://bakarimustafa.com',
+    social: {
+      twitter: 'realbakari',
+      github: 'realbakari',
+      blog: null,
+      linkedin: 'https://www.linkedin.com/in/realbakari/',
+    },
+    talks: [],
```

Comment:
Hey @realbakari thanks for your PR!
Could you please also add 1-2 talks please as well? Thanks!
Yuri

Reply:
Sure thing, thanks for the heads up mate.

Reply:
Hey @realbakari sorry for not getting to you for so long. I've got your second PR but it only duplication of this one. No talks were added, could you please add a few talks? Please see Vladimir Novick as an example.